libs/cardano-ledger-binary/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.3.0.0
 
+* Add `decodeNonEmptyList`
+* Expose `decodeCollection` and `decodeCollectionWithLen`
```

Comment:
These guys should stay hidden, they are a little bit too specific. It is safer to use `decodeList`
```suggestion
```